.github/workflows/verible-ci.yml

```diff
@@ -234,3 +235,37 @@ jobs:
         tag: ${{ env.TAG }}
         overwrite: true
         file_glob: true
+
+
+  MacOsBuild:
+    runs-on: macos-latest
+    steps:
+
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Tests
+      run: bazel test --test_output=errors --cxxopt=-Wno-range-loop-analysis ...
+
+
+  WindowsBuild:
+    runs-on: windows-latest
+    steps:
+
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Install dependencies
+      run: choco install llvm
+    # Windows support is in progress, only run "working" tests
+    # Remaining not working
+    #  - verilog/tool/lint : generated linter tests. Need adaption of bzl file
+    #  - verilog/tools/syntax/export_json_examples : python tests
```

Comment:
I think this comment can be removed now as in the meantime, all tests are passing on windows

Reply:
My bad. It's done now.